runtime/moonbase/tests/common/mod.rs

```diff
@@ -391,7 +391,7 @@ pub fn unchecked_eth_tx(raw_hex_tx: &str) -> UncheckedExtrinsic {
 
 pub fn ethereum_transaction(raw_hex_tx: &str) -> pallet_ethereum::Transaction {
 	let bytes = hex::decode(raw_hex_tx).expect("Transaction bytes.");
-	let transaction = rlp::decode::<pallet_ethereum::Transaction>(&bytes[..]);
+	let transaction = ethereum::EnvelopedDecodable::decode(&bytes[..]);
```

Comment:
@tgmichel to check if this si correct

Reply:
Looks good, either that or a more self-describing `<pallet_ethereum::Transaction as EnvelopedDecodeable>` is fine.